app/models/progress_document.rb

```diff
@@ -0,0 +1,7 @@
+class ProgressDocument < ApplicationRecord
+  belongs_to :commitment
+
+  has_one_attached :document
+
+  validates :document, attached: true, content_type: %w(application/msword application/pdf), size: { less_than: 25.megabytes }
```

Comment:
Different spacing to `commitment`, as noted above

Reply:
moved. not sure if this is right? happy to change it again if not